### PR TITLE
Dlg factor data frame

### PR DESCRIPTION
**Pull request by stevenndungu**
It was confusing though mixed  with exhaustion. You can be sure now understand it well. Thanks.

### CODE REVIEW

File: dlgFactorDataFrame.vb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿Imports Microsoft.VisualBasic
+
+Public Class Class1
+
+End Class
```
Comment (dannyparsons) [inline dlgFactorDataFrame.vb]: You still have this extra file here which should be deleted.

**Comment by stevenndungu** [comment]
@dannyparsons  this has now been deleted. 

Thanks

**Comment by dannyparsons** [comment]
Great, I will let you know when the R method is ready for you to implement.

**Comment by stevenndungu** [comment]
Nice, thanks.

**Pull request merged**